content/influxdb/cloud-dedicated/query-data/troubleshoot.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Troubleshoot responses and errors
+description: >
+  Learn how to handle responses and troubleshoot errors encountered when querying {{% cloud-name %}} with Flight+gRPC and Arrow Flight clients.
+weight: 201
+menu:
+  influxdb_cloud_dedicated:
+    name: Troubleshoot responses and errors
+    parent: Query data
+influxdb/cloud-dedicated/tags: [query, sql, influxql]
+alt_engine:
+---
+
+Learn how to handle responses and troubleshoot errors encountered when querying {{% cloud-name %}} with Flight+gRPC and Arrow Flight clients.
+
+<!-- TOC -->
+
+- [InfluxDB Flight responses](#influxdb-flight-responses)
+  - [Schema](#schema)
+    - [Example](#example)
+  - [InfluxDB error codes](#influxdb-error-codes)
+  - [Troubleshoot errors](#troubleshoot-errors)
+    - [Internal Error: Received RST_STREAM](#internal-error-received-rst_stream)
+    - [Internal Error: stream terminated by RST_STREAM with NO_ERROR](#internal-error-stream-terminated-by-rst_stream-with-no_error)
+    - [Invalid Argument Error: bucket <BUCKET_ID> not found](#invalid-argument-error-bucket-bucket_id-not-found)
+      - [Unauthenticated: Unauthenticated](#unauthenticated-unauthenticated)
+    - [Unauthenticated: read:<BUCKET_ID> is unauthorized](#unauthenticated-readbucket_id-is-unauthorized)
+
+## InfluxDB Flight responses
+
+{{% cloud-name %}} provides an InfluxDB-specific Arrow Flight service that uses gRPC Remote Procedure Calls (gRPC) to transport query result data in Arrow format.
```

Comment:
this is technically correct -- however I wonder if we want to highlight that it is (also) FlightSQL (which basically is a set of messages on top of Filght)?